explainer.md

```diff
@@ -0,0 +1,164 @@
+
+![OptMeowt Universal Setting](assets/images/OptMeowt_Universal_Setting.png)  
+The universal GPC setting of OptMeowt.
+
+![Firefox Universal Setting](assets/images/Firefox_Universal_Setting.png)  
+The universal setting of GPC in the Firefox browser settings.
+
+A user interface can show what response is at `https://{yourwebsite.com}/.well-known/gpc.json` and display that information to the users so they can understand what claims the website is making in terms of GPC compliance. This can be done regardless of the properties included on the JSON document, the main concern is the value of the `gpc` property, as seen here.
+
+![OptMeowt GPC Response](assets/images/OptMeowt_GPC_Response.png)  
+An example of how GPC responses can be surfaced (OptMeowt).
+
+### User-agents
+
+The above examples are from an extension in a web browser. User-agents should implement similar interface conventions. The authors of this document recommend that user-gents have some way to display to people the state of their GPC signal when it is on during the course of regular interaction with the site instead of putting it behind a settings page.
+
+A setting should be included to manage GPC. At the very least a simple Boolean-value switch should be available to the user within settings to manage GPC universally for the user-agent.
+
+If GPC is not turned on by default, there should be a control that is accessible to people to turn it on when interacting with an individual site.
+
+If the user-agent makes the GPC setting visible when active, it should retain individuals’ privacy choices when they turn off GPC for a specific site.
+
+User-agents should not challenge people with a request to set GPC in either mode beyond initial setup. Per-domain settings of GPC should be up to an individual to engage with, not pushed via a notification, modal, pop-up, or similar interactive element.
+
+### Adopting on Your Websites
+
+Given the complexities of existing privacy choice and consent frameworks, sites that implement GPC should disclose how they treat it in any jurisdiction for which they adopt it and how they deal with conflicts between a GPC signal and other specific privacy choices that an individual has already made directly with the site, including instances where third party sharing may be permitted, such as sharing to service providers/processors or at the direction of the individual.
+
+Where industry standards set specific strings or signals that are needed to communicate people’s privacy choices, sites should anticipate translating GPC into the downstream signal. A good example of such setting is handling GPC to set California-based US Privacy Strings for advertising technology:
+
+![USPAPI](assets/images/USPAPI.png)  
```

Comment:
You might be able to do something like this instead:

`````suggestion
```javascript
if (whatever) {
  do_whatever();
}
```
`````

Reply:
That is a good point! I updated the PR. Obliged, @martinthomson!